File: Calculator_CPP_Python/Calculator_CPP/Calculator.cpp
#include <iostream>

using namespace std;

int main()
{

	double a = 0;

	double b = 0;

	double result = 0;

	double choice = 0;

	cout <<"Enter the number a : ";
	cin >> a;
	cout << endl;
	cout << "Enter the number b : ";
	cin >> b;
	cout << endl;

	choise_1:

	cout << endl << " | Choose an action :  \n 1 Addition \n 2 Subtraction \n 3 Multiplication \n 4 Division \n \n";

	cin >> choice;

	if (choice == 1)
	{
		result = a + b;
	}
	else if(choice == 2)
	{
		result = a - b;
	}
	else if (choice == 3)
	{
		result = a * b;
	}
	else if (choice == 4)
	{
		result = a / b;
	}
	else
	{
		cout << endl << " |  This action is not listed \n" << endl;
		goto choise_1;
	}

	cout << endl << " |  Result = " << result << endl;

	cout << endl << "..." << endl;
	
	double Stop_prog; cin >> Stop_prog;

	return 0;

}

File: Calculator_CPP_Python/Calculator_Python/Calculator.py
a = int (input('Enter the number a: '))
b = int (input('Enter the number b: '))

choice = int (input("\n" + " |  Choose an action: " + " \n 1 Addition \n 2 Subtraction \n 3 Multiplication \n 4 Division \n \n"))

if choice == 1:

    result = a + b
    print ("\n" + " |  Result = " + str(result) + "\n")

elif choice == 2:

    result = a - b
    print ("\n" + " |  Result = " + str(result) + "\n")

elif choice == 3:

    result = a * b
    print ("\n" + " |  Result = " + str(result) + "\n")

elif choice == 4:

    result = a / b
    print ("\n" + " |  Result = " + str(result) + "\n")

else:
	print ("\n" + " |  This action is not listed \n")

input("Press \"Enter\" to complete the program...")